js/src/common/models/AccessToken.js

```diff
@@ -0,0 +1,19 @@
+import Model from '../Model';
+
+export default class AccessToken extends Model {
+  apiEndpoint() {
```

Comment:
Is there a reason why we don't want to just call the model type 'tokens'?

Reply:
Yes. Because there's also `email_tokens`, `password_tokens` and `registration_tokens`. I feel like calling it `token` is just a recipe for confusion.

Reply:
Agreed. In that case, should we change the API controller path?